curvefs/src/mds/topology/topology_item.h

```diff
@@ -497,6 +497,7 @@ class CopySetInfo {
           epoch_(v.epoch_),
           peers_(v.peers_),
           partitionNum_(v.partitionNum_),
+          partitionIds_(v.partitionIds_),
```

Comment:
The partitionNum_ is not needed. In GetPartitionNum can return partitionIds_.size().

Reply:
If this is removed, the software will have problems with backward compatibility.

Reply:
done, i have removed partitionNum_.